src/Card/CardDeck.java

```diff
@@ -32,6 +35,29 @@ public CardDeck(int easyCardNumber, int hardCardNumber, int harderCardNumber){
         this.easyCardNumber = easyCardNumber;
         this.hardCardNumber = hardCardNumber;
         this.harderCardNumber = harderCardNumber;
+        studying = false;
+    }
+    
+    // when the user wants the program to ask the words
+    public void letsStudying(){
```

Comment:
I think `letsStudy` would be the appropriate name for this function according to English grammer rules.